Libplanet.Extensions.Cocona/Commands/ApvCommand.cs

```diff
@@ -263,7 +267,32 @@ void TreeIntoTable(IValue tree, List<(string, string)> table, string key)
                 TreeIntoTable(v.Extra, data, "extra");
             }
 
-            Utils.PrintTable(("Field", "Value"), data);
+            if (json)
+            {
+                Console.WriteLine(JsonSerializer.Serialize(data.ToDictionary(e => e.Item1, e => e.Item2)));
+            }
+            else
+            {
+                Utils.PrintTable(("Field", "Value"), data);
+            }
+        }
+
+        [Command(Description = "Query app protocol version (a.k.a. APV) of target node.")]
+        public void Query(
+            [Argument(
+                Name = "TARGET",
+                Description = "Comma seperated peer information of target node.")]
```

Comment:
It would be better if it has an input example so that users can guess its format.

Reply:
Applied d1e909cf8bb2e82770d03cb3e3ac24bb86ce5d89